src/System.Device.Gpio/System/Device/Gpio/AlternatePinMode.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
```

Comment:
copyright header missing

Reply:
All of these done (i think)

Reply:
Ups, forgot to check the files in the System assembly.